wiki/backup.md

```diff
@@ -0,0 +1,85 @@
+# Backing up data
+
+This page aims to explain how to back up your data to either your cloud or an external drive, why you should regularly back up your files.
+
+## What do you need to back up?
+
+It mostly depends on your preference, but you can start with your personal documents, photos, videos. These should be backed up regularly in case you lost them.
+
+## Why do I need to back up my files?
+
+Some updates, software bugs or malware could delete your data, your computer's drive could fail at some point, taking backups is useful when you regularly modify system files or when you format your disks frequently.
+
+It's safer to store data in multiple places in case something happened to your computer's drive.
+
+## The 3-2-1 Backup rule
+
+The 3-2-1 rule is keeping 3 copies of your data, two on different storage mediums (your computer's drive and a backup drive for example) and one on the cloud.
+
+For instance, if you have File X on your computer's main drive, you have to create another copy to put on your backup drive and/or on the cloud. Now you have 3 copies of File X.
+
+## Backup locally
+
+### Using an external drive
+
+If you have an external drive, you could use it to backup your files by transferring your files to it. Your external drive could also include a backup software made by the drive's manufacturer, you can use that to back up your files.
+
+### Using File History (Windows 10)
+
+File History is a tool which automatically makes backups of your files. You can keep track of each file's revision by using the Previous Versions tab in a file's properties window or by right clicking on a file then selecting "Restore previous versions", and you can also choose when to back up files and when to remove the backups.
+
+![File History page](img/backup/file-history-page.png)
+
+![File History settings](img/backup/file-history-settings.png)
+
+#### Add a drive to File History
+
+- Open Settings.
+- Go to Update & Security > Backup.
+- Add a drive to File History in order to use as a backup drive.
+
+### Using System Image Backup
+
+You can use System Image Backup to back up your system image. A system image is basically an image of the current state of the operating system stored in a file. You can use these system images to revert the system to a previous state.
+
+![System Image Backup option](img/backup/system-image-backup.png)
+
+- Search "File History" or go to Settings > Update & Security > Backup and click on "Go to Backup and Restore (Windows 7)" (Windows 10).
+- On the Control Panel sidebar, click on "System Image Backup".
+![](img/backup/create-a-system-image.png)
+- Click on "Create a system image" and follow the instructions.
+![](img/backup/create-system-image-wizard.png)
+
+## Backup to a cloud service
+
+You can also back up your data on a cloud service (for example: OneDrive, Google Drive etc...), this solution could be better in case you lost your backup drive.
```

Comment:
```suggestion
You can also back up your data on a cloud service (for example: OneDrive, Google Drive etc...), this solution could be better in case you lose your backup drive.
```